src/main/drivers/serial_uart_stm32f7xx.c

```diff
@@ -332,6 +334,12 @@ void uartGetPortPins(UARTDevice_e device, serialPortPins_t * pins)
     }
 }
 
+void uartClearIdleFlag(uartPort_t *s)
```

Comment:
This function doesn't seem to be ever used.

Reply:
Yeah, was not sure if I should leave this out or not even if unused since F7's seem to use HAL and F3's/F4's don't. If we dont want it though, no problem removing it.

Reply:
If theres nothing else, I can remove this function for F7s and squash the commits too while I'm at it

Reply:
Don't see anything else, thanks!

Reply:
updates have been pushed!